src/components/forms/ExportFormModal.vue

```diff
@@ -25,7 +25,11 @@
         </b-button>
       </template>
 
-      <b-form>
+      <b-form @submit='
```

Comment:
Change @submit to @submit.prevent
Don't need to create a new PR, just add a new commit for this one
Thanks